server/controllers/stock/core.js

```diff
@@ -403,6 +403,50 @@ function getStockConsumption(periodIds) {
   return db.exec(sql, [periodIds]);
 }
 
+
+/**
+ * @function getDailyStockConsumption
+ *
+ * @description returns the daily (periodic) stock consumption (CM)
+ *
+ * @param {array} periodIds
+ */
+async function getDailyStockConsumption(params) {
+  params.consumption = true;
+
+  if (params.period_id) {
+    const period = await db.one('SELECT start_date,end_date FROM period WHERE id=?', params.period_id);
```

Comment:
Oh la la.  We don't have the period_id in stock_movements?  That is a problem.

Made an issue: https://github.com/IMA-WorldHealth/bhima/issues/4347